Kinvey/js-sdk
# MLIBZ-2316: Server Side Delta Set

Add new server side delta set functionality.

#### Changes
- Updated `request/deltaFetch.js` to perform server side delta set requests.

#### Tests
- Verify `request/deltaFetch.js` sends correct requests.
- Verify `datastore/cachestore.js` behaves correctly when `useDeltaFetch` is turned on.
